cli/cmd/report_definitions_diff.go

```diff
@@ -0,0 +1,166 @@
+//
+// Author:: Darren Murray(<darren.murray@lacework.net>)
+// Copyright:: Copyright 2023, Lacework Inc.
+// License:: Apache License, Version 2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package cmd
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/fatih/color"
+	"github.com/pkg/errors"
+	"github.com/pmezard/go-difflib/difflib"
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v3"
+)
+
+// reportDefinitionsDiffCommand command is used to compare 2 lacework report definition versions
+var reportDefinitionsDiffCommand = &cobra.Command{
+	Use:   "diff <report_definition_id> <version> <version>",
+	Short: "Compare 2 versions of a report definition",
+	Long: `Compare 2 versions of a report definition.
+
+To see a diff of 2 report definition versions:
```

Comment:
Run this by the docs team but, I think we should use numbers here.
```suggestion
	Short: "Compare two versions of a report definition",
	Long: `Compare two versions of a report definition.

To see a diff of two report definition versions:
```